releng/tools.vitruv.applications.cbs.parent/pom.xml

```diff
@@ -31,6 +31,54 @@
 	</repositories>
 
 	<profiles>
+		<profile>
+			<id>nightly-update-site-framework</id>
+			<activation>
+     			<property>
+        			<name>!vitruv.framework.path</name>
+      			</property>
+      		</activation>
```

Comment:
The indentation seems to be faulty for all `activation` blocks.

Reply:
Fixed in https://github.com/vitruv-tools/Vitruv-Applications-ComponentBasedSystems/pull/188/commits/7ce6f4939cc11d656d455ac62651ce3dec04754d